From 6911dbdc62c389e73a9e1e4d5eb8ba1af2358190 Mon Sep 17 00:00:00 2001
From: Glenn Washburn <development@efficientek.com>
Date: Wed, 11 May 2022 22:19:44 -0500
Subject: [PATCH 217/261] tests: Show host determined fs UUID when hfs UUID
 test fails

On failure, the hfs test should show both the host and GRUB determined fs
UUID. Prior to this change, both outputs where generated by GRUB, which is
less helpful in determining the cause of failure.

Signed-off-by: Glenn Washburn <development@efficientek.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
---
 tests/util/grub-fs-tester.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/util/grub-fs-tester.in b/tests/util/grub-fs-tester.in
index 3c1d712de..43f6175c3 100644
--- a/tests/util/grub-fs-tester.in
+++ b/tests/util/grub-fs-tester.in
@@ -1353,6 +1353,7 @@ for LOGSECSIZE in $(range "$MINLOGSECSIZE" "$MAXLOGSECSIZE" 1); do
 			:
 		    else
 			echo UUID FAIL
+			echo "$FSUUID"
 			echo "$LSOUT"
 			echo "$GRUBUUID"
 			for lodev in $LODEVICES; do
-- 
2.36.1

